### PR TITLE
Included Small Developer's Guide to ReadME

### PR DESCRIPTION
**Pull request by jainishmehta**
Simple change to allow new developers better integrate repo into VSCode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,13 @@ Sample setting:
 ```
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
+## Developer's Guide
+
```
Comment (wmdietl) [inline README.md:106]: At the moment CI does `yarn && yarn compile`
https://github.com/eisopux/checker-framework-vscode/blob/master/azure-pipelines.yml#L31

Is that needed in addition to `vsce package` or should CI be changed?

File: README.md [hunk 2]
```diff
@@ -102,6 +102,13 @@ Sample setting:
...
+To install the extension on VSCode:
+1. Install its dependencies (`npm install`) and `npm install -g vsce`.
```
Comment (wmdietl) [inline README.md:108]: Should this use yarn instead of npm?

File: README.md [hunk 3]
```diff
@@ -102,6 +102,13 @@ Sample setting:
...
+2. In the root of the project, run `vsce package`to generate a .vsix file and then do
+`code --install-extension my-extension-0.0.1.vsix`, replacing my-extension-0.0.1.vsix to the .vsix file recently created. 
+Under View -> Extensions on VSCode check if the 'checker-framework-language-server' is installed and if not, search for it and install it.
```
Comment (wmdietl) [inline README.md:111]: Should this be a 3rd step?
Does "search for it" refer to a file search or something else?

File: README.md [hunk 4]
```diff
@@ -102,6 +102,13 @@ Sample setting:
 ```
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
+## Developer's Guide
+
+To install the extension on VSCode:
+1. Install its dependencies (`npm install`) and `npm install -g vsce`.
+2. In the root of the project, run `vsce package`to generate a .vsix file and then do
```
Comment (wmdietl) [inline README.md:109]: Split this up into a step that builds and a separate step that installs.

**Review by wmdietl — COMMENTED** [review]
Thanks for the improvements!

File: README.md (hunk 3 above)
Comment (jainishmehta) [reply README.md:111]: Sorry, yeah "search for it" is just a file search, I'll update this to be more clear

File: README.md (hunk 1 above)
Comment (jainishmehta) [reply README.md:106]: I'll have to reinstall the extension, I did the steps I described here, but it is very likely that this CI does it already, in which case this is just extra information

**Comment by wmdietl** [comment]
Superseded by #84

**Pull request closed without merging**